src/FNA3D_Driver_D3D11.c

```diff
@@ -2689,7 +2703,10 @@ static void D3D11_INTERNAL_CreateBackbuffer(
 	}
 
 	/* Create a render target view for the swapchain */
-	swapchainViewDesc.Format = DXGI_FORMAT_R8G8B8A8_UNORM;
+	DXGI_FORMAT swapchainFormat = (actualDxgiFormat == DXGI_FORMAT_R8G8B8A8_UNORM_SRGB)
```

Comment:
Can remove this local variable, plus it probably breaks on some C compilers

Reply:
Oh are ternary expressions not available in classic C?

Reply:
They're available, but variables mid-function aren't supported by older MSVC releases. I think you can just assign this directly to the struct member right below.